From 3016a7beb0056fb4fa4666cfa2f3a6dc4d055e09 Mon Sep 17 00:00:00 2001
From: Stuart Morgan <smorgan@mythtv.org>
Date: Thu, 18 Dec 2014 19:25:25 +0000
Subject: [PATCH] Copy m_added and m_created in the MythNotificationScreen
 copy operator instead of initialising them to false. May
 fix some reported issues with performance during playback
 and duplicate popups.

---
 mythtv/libs/libmythui/mythnotificationcenter.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mythtv/libs/libmythui/mythnotificationcenter.cpp b/mythtv/libs/libmythui/mythnotificationcenter.cpp
index 5d5a947..c55f41d 100644
--- a/mythtv/libs/libmythui/mythnotificationcenter.cpp
+++ b/mythtv/libs/libmythui/mythnotificationcenter.cpp
@@ -263,8 +263,8 @@ MythNotificationScreen &MythNotificationScreen::operator=(const MythNotification
 
     m_update        = m_content; // so all fields are initialised regardless of notification type
 
-    m_added         = s.m_added;
-    m_created       = s.m_created;
+    m_added         = false;
+    m_created       = false;
 
     m_artworkImage     = NULL;
     m_titleText        = NULL;
-- 
1.7.10.2

